src/Compilers/CSharp/csc/project.json

```diff
@@ -4,7 +4,9 @@
     "net45": { }
   },
   "runtimes": {
+    "win7": { },
     "win7-anycpu": { },
```

Comment:
I'm leaving them in for compat with RTM until we declare RTM no longer suitable for Roslyn development. More practically, our build machines still need that!